src/Console/Deployer.php

```diff
@@ -76,7 +76,11 @@ public function deploy(SymfonyStyle $io, bool $dryRun) : void
         if (!$dryRun) {
             $progress->setMessage('Uploading the lambda');
             $progress->display();
-            $process = new Process('serverless deploy', '.bref/output');
+            $serverlessCommand = 'serverless deploy';
+            if (null !== $stage) {
+                $serverlessCommand .= ' --stage ' . $stage;
```

Comment:
We could use `escapeshellarg()` just in case? (I don't know if there can be special characters in stages but it doesn't cost much to use it)